client/test/features/specs/support/clean.state.js

```diff
@@ -238,6 +237,21 @@ module.exports = {
         expand: null,
         scrollOffset: 0,
         sortDescending: true,
+      },
+
+      nrInfo: {
```

Comment:
This was missing and caused the transactions tests (and code under test) to fail.

Reply:
Good catch